servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/PublisherFlatMapMergeTest.java

```diff
@@ -190,6 +190,35 @@ private void singleConcurrencyComplete(Publisher<Integer> flatMappedPublisher) {
         subscriber.awaitOnComplete();
     }
 
+    @Test
+    void singleConcurrencyAsync() {
```

Comment:
This change is not related to the PR

Reply:
yes, inclusion in this PR was unintended.